### PR TITLE
Refactor/dedicated univers components

### PR DESCRIPTION
**Pull request by radireddy**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #
| License          | BSD 3-Clause

## Description

MOved all common code and components that are used in billing and account module into dedicated-universe-module

### CODE REVIEW

File: packages/manager/modules/dedicated-universe-components/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,419 @@
+## [2.0.3](https://github.com/ovh/manager/compare/@ovh-ux/ng-ovh-cloud-universe-components@2.0.2...@ovh-ux/ng-ovh-cloud-universe-components@2.0.3) (2020-08-18)
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/CHANGELOG.md:1]: Changelog should be empty if it's a new module. Looks like it was copy/pasted from `cloud-universe-components`. Any issue with using the sao module generator ?

File: packages/manager/modules/dedicated-universe-components/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@ovh-ux/ng-ovh-dedicated-universe-components",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Collection of components for the Dedicated Cloud.",
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/package.json:5]:
```suggestion
  "description": "Collection of components for the Dedicated universe.",
```

File: packages/manager/modules/dedicated-universe-components/package.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    "@ovh-ux/ng-ovh-http": "^5.0.0",
+    "@ovh-ux/ng-ovh-user-pref": "^2.0.0",
+    "@ovh-ux/ng-translate-async-loader": "^2.1.0",
+    "@ovh-ux/ui-kit": "^4.2.3",
+    "@uirouter/angularjs": "^1.0.23",
+    "angular": "^1.7.5",
+    "angular-translate": "^2.18.1",
+    "angular-ui-bootstrap": "~1.3.3",
+    "ovh-api-services": "^9.39.1",
+    "regenerator-runtime": "^0.13.7"
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/package.json:48]: Why is this needed as a peer ? Isn't it just needed in the dedicated app ?

File: packages/manager/modules/dedicated-universe-components/src/components/user/user.service.js [hunk 4]
```diff
@@ -0,0 +1,195 @@
+import flatten from 'lodash/flatten';
+
+import { User } from '@ovh-ux/manager-models';
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/src/components/user/user.service.js:3]: didn't seem declare as a dependency

File: packages/manager/modules/dedicated-universe-components/src/components/index.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+
+import user from './user';
+import noDefaultMeansOfPayment from './no-default-means-of-payment';
+import stepper from './stepper';
+import OvhHttp from './ovh-angular-http/ovh-angular-http.module';
+
+const moduleName = 'ducComponents';
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/src/components/index.js:8]:
```suggestion
const moduleName = 'dedicatedUniverseComponents';
```

File: packages/manager/modules/dedicated-universe-components/src/components/index.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+
+import user from './user';
+import noDefaultMeansOfPayment from './no-default-means-of-payment';
+import stepper from './stepper';
+import OvhHttp from './ovh-angular-http/ovh-angular-http.module';
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/src/components/index.js:6]: Why not import it in the submodule that needs the dependency ? It doesn't seem needed at the root level

File: packages/manager/modules/dedicated-universe-components/src/components/no-default-means-of-payment/index.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import component from './no-default-means-of-payment.component';
+
+const moduleName = 'ovhManagerNoDefaultMeansOfPaymentModule';
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/src/components/no-default-means-of-payment/index.js:3]:
```suggestion
const moduleName = 'ducNoDefaultMeansOfPayment';
```

File: packages/manager/modules/dedicated-universe-components/src/config/constants.config.js [hunk 8]
```diff
@@ -0,0 +1,1333 @@
+import defaults from 'lodash/defaults';
+
+const constants = {
+  EU: {
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/src/config/constants.config.js:4]: I don't know if constants should be extracted from their universes in a module with components specific to a universe

File: packages/manager/modules/dedicated-universe-components/src/config/index.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
+import { Environment } from '@ovh-ux/manager-config';
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/src/config/index.js:1]: Dependency not declared

File: packages/manager/modules/dedicated-universe-components/src/config/index.js [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+import constantsConfig from './constants.config';
+
+const aapiRootPath = '/engine/2api/';
+const swsProxyRootPath = 'apiv6/';
+
+const target = Environment.getRegion();
+// const prodMode = !!WEBPACK_ENV.production;
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/src/config/index.js:8]: If webpack variable are used maybe we should keep those declaration in the app

File: packages/manager/modules/dedicated-universe-components/src/index.js [hunk 11]
```diff
@@ -0,0 +1,96 @@
+import angular from 'angular';
+import 'regenerator-runtime/runtime';
```
Comment (marie-j) [inline packages/manager/modules/dedicated-universe-components/src/index.js:2]: This should be imported in the app not the module. The app should be the one choosing to polyfill or not

**Review by marie-j — COMMENTED** [review]
No changes made to the dedicated app ?

File: packages/manager/modules/dedicated-universe-components/README.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+## Install
+
+```sh
+yarn add @ovh-ux/ng-ovh-dedicated-universe-components
+```
+## Usage
+
+```js
+import angular from 'angular';
+import ngOvhCloudUniverseComponents from '@ovh-ux/ng-ovh-dedicated-universe-components';
```
Comment (JeremyDec) [inline packages/manager/modules/dedicated-universe-components/README.md:16]:
```suggestion
import ngOvhDedicatedUniverseComponents from '@ovh-ux/ng-ovh-dedicated-universe-components';
```

File: packages/manager/modules/dedicated-universe-components/src/config/constants.config.js (hunk 8 above)
Comment (varun257) [reply packages/manager/modules/dedicated-universe-components/src/config/constants.config.js:4]: These constants are used in multiple modules. What do you suggest we do ?

**Comment by ovh-cds** [comment]
CDS Report test#16563.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

**Comment by ovh-cds** [comment]
CDS Report test#16622.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

File: packages/manager/modules/dedicated-universe-components/src/config/index.js (hunk 10 above)
Comment (radireddy) [reply packages/manager/modules/dedicated-universe-components/src/config/index.js:8]: yes, I will add this declaration into the app.

**Comment by ovh-cds** [comment]
CDS Report test#16658.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

File: packages/components/ng-ovh-contacts/CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,12 @@
+## [4.0.2](https://github.com/ovh/manager/compare/@ovh-ux/ng-ovh-contacts@4.0.1...@ovh-ux/ng-ovh-contacts@4.0.2) (2020-10-28)
```
Comment (marie-j) [inline packages/components/ng-ovh-contacts/CHANGELOG.md:1]: A rebase might be needed

**Comment by ovh-cds** [comment]
CDS Report test#16764.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

**Comment by ovh-cds** [comment]
CDS Report test#16806.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

**Comment by ovh-cds** [comment]
CDS Report test#16843.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

**Comment by ovh-cds** [comment]
CDS Report test#16867.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔
